src/Role_Command.php

```diff
@@ -253,26 +252,28 @@ public function reset( $args, $assoc_args ) {
 
 		self::persistence_check();
 
-		if ( ! \WP_CLI\Utils\get_flag_value( $assoc_args, 'all' ) && empty( $args ) )
-			WP_CLI::error( "Role key not provided, or is invalid." );
+		if ( ! \WP_CLI\Utils\get_flag_value( $assoc_args, 'all' ) && empty( $args ) ) {
+			WP_CLI::error( 'Role key not provided, or is invalid.' );
+		}
 
 		if ( ! function_exists( 'populate_roles' ) ) {
-			require_once( ABSPATH.'wp-admin/includes/schema.php' );
+			require_once ABSPATH . 'wp-admin/includes/schema.php';
 		}
 
 		global $wp_roles;
-		$all_roles = array_keys( $wp_roles->roles );
+		$all_roles     = array_keys( $wp_roles->roles );
 		$preserve_args = $args;
 
 		// Get our default roles.
-		$default_roles = $preserve = array( 'administrator', 'editor', 'author', 'contributor', 'subscriber' );
-		$before = array();
+		$default_roles = [ 'administrator', 'editor', 'author', 'contributor', 'subscriber' ];
+		$preserve      = [ 'administrator', 'editor', 'author', 'contributor', 'subscriber' ];
+		$before        = array();
```

Comment:
Stick to one notation.
```suggestion
		$before        = [];
```